prysm.sh

```diff
@@ -71,6 +71,8 @@ if [ "$#" -lt 1 ]; then
     exit 1
 fi
 
+cat < /dev/null > /dev/tcp/8.8.8.8/53;
```

Comment:
Just realized this most likely wont work at all. Been trying to look for good methods but none of them have worked.